appregistry/cmd/details.go

```diff
@@ -29,21 +29,24 @@ var (
 
 // NewDetailsCmd creates a new details command that shows the details of an ignite application repository.
 func NewDetailsCmd() *cobra.Command {
-	return &cobra.Command{
-		Use:     "details [app name]",
+	c := &cobra.Command{
+		Use:     "details [app slug]",
```

Comment:
I find "app name" better to understand for users.

Reply:
wdyt rename slug to ID?

Reply:
yes that sounds better